### PR TITLE
update for proxy release-1.3

### PR DESCRIPTION
**Pull request by bdecoste**

### CODE REVIEW

**Review by brian-avery — APPROVED** [review]

File: istio-proxy/fetch.sh [hunk 1]
```diff
@@ -265,13 +277,13 @@ function patch_class_memaccess() {
 function replace_ssl() {
   if [ "$REPLACE_SSL" = "true" ]; then
     pushd ${PROXY_FETCH_DIR}/proxy
-      git clone http://github.com/maistra/istio-proxy-openssl -b ${OPENSSL_GIT_BRANCH}
+      git clone http://github.com/bdecoste/istio-proxy-openssl -b ${OPENSSL_GIT_BRANCH}
```
Comment (dmitri-d) [inline istio-proxy/fetch.sh:280]: hmm. I think this should be pointing to https://github.com/Maistra/istio-proxy-openssl

File: istio-proxy/fetch.sh [hunk 2]
```diff
@@ -265,13 +277,13 @@ function patch_class_memaccess() {
...
       pushd istio-proxy-openssl
         ./openssl.sh ${PROXY_FETCH_DIR}/proxy OPENSSL
       popd
       rm -rf istio-proxy-openssl
 
-      git clone http://github.com/maistra/envoy-openssl -b ${OPENSSL_GIT_BRANCH}
+      git clone http://github.com/bdecoste/envoy-openssl -b ${OPENSSL_GIT_BRANCH}
```
Comment (dmitri-d) [inline istio-proxy/fetch.sh:286]: ditto

File: istio-proxy/fetch.sh (hunk 1 above)
Comment (dmitri-d) [reply istio-proxy/fetch.sh:280]: ugh, Bill's 1.1 branch has 1.3-related commits, which will need to be moved to Maistra/istio-proxy-openssl

**Review by dmitri-d — CHANGES_REQUESTED** [review]
There are references to Bill's istio-proxy-openssl repository here. Bill's repository contains commits that need to be moved to Maistra/istio-proxy-openssl.

**Review by dmitri-d — APPROVED** [review]

**Comment by dmitri-d** [comment]
Approving the changes, as it would be easier (process-wise) to make the changes in the comments after this PR has been merged in

**Pull request merged**